site-src/index.md

```diff
@@ -20,54 +20,119 @@ variety of networking use-cases.
+build on top of many implementations more easily, and it reduces the need for
+Gateway implementers to build custom (yet oddly similar) APIs. There are
+diverse benefits for both the users and implementers of a portable and
+standardized API. However, deep and meaningful portability is challenging.
+Portability is a constant choice between consistent behavior and the unique &
+valuable capabilities of a specific implementation - how to provide an API
+that does both? 
+
+The Gateway API is designed to strike a balance between these two sides
+through flexible conformance and extensibility. Gateway defines [multiple
+levels of conformance](guidelines.md#conformance), which provides implementers
+with flexibility in how much of the Gateway API they want to support. In
+combination with various extension points within the API, this allows the core
+functionality of Gateway to be portable, while still surfacing
+implementation-specific capabilities.
+
+### Who is working on Gateway?
+
+The Gateway API is a Kubernetes
+[SIG-Network](https://github.com/kubernetes/community/tree/master/sig-network)
+project being built by a variety of companies and users. Current and
+in-progress implementations include Contour, Google Kubernetes Engine (GKE),
+Istio, Kong, and Traefik. Check out the [implementations
+reference](implementations.md) to see the latest projects & products that
+support Gateway. If you are interested in contributing to or building an
+implementation using the Gateway API then don’t hesitate to [get
+involved!](#how-to-get-involved)
 
 [sig-network]: https://github.com/kubernetes/community/tree/master/sig-network
 
-### Where to get started
-
-To get started, please read through [API overview](api-overview.md). These
-documents give the necessary background to understand the API and the use-cases
-it targets. Once you have a good understanding of the API at a higher-level,
-please follow one of our [guides](guides.md) to dive deeper into different parts
-of the API.
+### Getting started
 
-For a complete API reference, please refer to:
+Whether you are a user interested in using the Gateway API  
+or an implementer interested in conforming to the API, the following resources 
+will help give the necessary background to understand the API:
 
-- [API reference](spec.md)
-- [Go docs for the package](https://pkg.go.dev/sigs.k8s.io/gateway-api/apis/v1alpha1)
+- Start with the [API overview](api-overview.md) to understand the basics
+- Follow through the [user guides](guides.md) to see what use-cases Gateway 
+addresses
+- Check out the [Gateway controller implementations](implementations.md) to 
+get hands-on experience in your Kubernetes cluster
+- See the [API reference](spec.md) for a detailed breakdown of the API surface 
+area
+- If you’re ready to start developing with Gateway, check out the 
+[Go docs for the package](https://pkg.go.dev/sigs.k8s.io/gateway-api/apis/v1alpha1)
```

Comment:
This is a huge improvement! I think this may be one of the most helpful parts of the page, I just hope it doesn't get missed due to it being so far down the page.